encoders/firebase-decoders-json/build.gradle

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 Google LLC
```

Comment:
nit: this file needs to be called `fireabse-decoders-json.gradle` not `build.gradle`(this is a special setup we have in this repo).